Arrowgene.Ddon.Shared/AssetRepository.cs

```diff
@@ -42,7 +42,9 @@ public class AssetRepository
         public const string JobValueShopKey = "JobValueShop.csv";
         public const string StampBonusKey = "StampBonus.csv";
         public const string SpecialShopKey = "SpecialShops.json";
-        public const string PawnCostReductionKey = "PawnCostReduction.json"; 
+        public const string PawnCostReductionKey = "PawnCostReduction.json";
+        public const string PawnCraftSkillCostRateKey = "PawnCraftSkillCostRate.csv";
```

Comment:
I don't understand why we keep moving things from json back to CSV. CSV is a terrible format.

Reply:
It's not great for your dataset. You already have rows with a bunch of empty commas which implies there is optional fields. I also need to use another tool which specializes in spreadsheets to view it in any sane manner.